### PR TITLE
CORE-1482 Add Featured Apps category to Apps view

### PR DESCRIPTION
**Pull request by ash3rz**
image

image

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default {
     APPS_UNDER_DEV: "Apps under development",
     FAV_APPS: "Favorite Apps",
     MY_PUBLIC_APPS: "My public apps",
+    FEATURED_APPS: "Featured Apps",
     HPC: "High-Performance Computing",
     BROWSE_ALL_APPS: "Browse All Apps",
     BROWSE_ALL_APPS_ID: "pppppppp-pppp-pppp-pppp-pppppppppppp",
```
Comment (sri2k1us) [inline src/constants.js:22]: While we are in it, can we fix the casing. For example, `Apps under development` - > `Apps Under Development`, `My public apps` -> `My Public Apps` etc..

**Review by sri2k1us — COMMENTED** [review]
I think Nirav / Tyson wanted `Featured Apps` to be the default selection. Can you please check with them ?

File: src/components/apps/toolbar/AppNavigation.js [hunk 2]
```diff
@@ -200,6 +202,10 @@ function AppNavigation(props) {
         setAnchorEl(null);
     };
 
+    const categoryI18nName = (name) => {
+        return name.toLowerCase().replaceAll(" ", "_");
+    };
```
Comment (psarando) [inline src/components/apps/toolbar/AppNavigation.js:207]: ✨

**Review by psarando — APPROVED** [review]
LGTM 👍

**Review by sri2k1us — APPROVED** [review]
LGTM 🚢

File: src/constants.js [hunk 3]
```diff
@@ -16,9 +16,11 @@ export default {
...
+    FEATURED_APPS_ID: "cccccccc-cccc-cccc-cccc-cccccccccccc",
```
Comment (ash3rz) [inline src/constants.js:23]: Wasn't sure if this should be in a config instead - I'll gladly add it if that's the preference

Comment (psarando) [reply src/constants.js:23]: My instinct is to say that it should be in a config, but we already have `BROWSE_ALL_APPS_ID` as a constant. Plus I don't think there's a way to update configs without rebuilding the entire UI right now anyway 🤷

**Review by psarando — APPROVED** [review]
LGTM 👍

**Pull request merged**